internal/gatewayapi/testdata/merge-valid-multiple-gateways-multiple-routes.out.yaml

```diff
@@ -216,6 +216,7 @@ infraIR:
       metadata:
         labels:
           gateway.envoyproxy.io/owning-gatewayclass: envoy-gateway-class
+          gateway.networking.k8s.io/gateway-name: gateway-1,gateway-2
```

Comment:
All gateway names are concatenated and stored in a single label if mergeGateways is enabled.
 Probably not the best way to do this, should revisit this and come up with a better solution upstream.